frontend_tests/casper_tests/03-narrow.js

```diff
@@ -180,7 +180,7 @@ function search_silent_user(str, item) {
         casper.waitUntilVisible('#silent_user', function () {
             casper.test.info("Empty feed for silent user visible.");
             var expected_message = "\n        You haven't received any messages sent by this user yet!"+
-                                    "\n      ";
```

Comment:
I still couldn't figure out why the changes to indentation here caused changes in spacing after these messages. It is not the case we tampered with the message div block or para block.
For the moment I just removed the space from the tests as looking at stuff being rendered it didn't felt like there was any reason not to do this.

Reply:
I think it's just a sloppy test; this should be fine.